src/components/Facet.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <dl class="facet">
-    <dt>{{ facetHeader }}</dt>
+    <dt v-if="facetList && facetList.length">{{ facetHeader }}</dt>
```

Comment:
Hmmm. Should this condition actually be on the `dl` and not the `dt` to suppress the whole list?

Reply:
Oh, never mind as I see you have tests that show them not showing up like this.